docs/src/api_bindings.md

```diff
@@ -477,6 +477,7 @@ h5pl_size
 - [`h5p_get_fapl_mpio32`](@ref h5p_get_fapl_mpio32)
 - [`h5p_get_fapl_mpio64`](@ref h5p_get_fapl_mpio64)
 - [`h5p_get_fapl_multi`](@ref h5p_get_fapl_multi)
+- [`h5p_get_fapl_ros`](@ref h5p_get_fapl_ros)
```

Comment:
Hmm... I think we may need to the patch the generator.

Reply:
agreed. the name should be `h5p_get_fapl_ros3`

Reply:
I added a bind exception for ros3 in cc6625e